### PR TITLE
More model tweaks

### PR DESCRIPTION
**Pull request by adamcik**

### CODE REVIEW

File: mopidy/models.py [hunk 1]
```diff
@@ -145,11 +147,12 @@ def __new__(cls, name, bases, attrs):
                 value._name = key
 
         attrs['_fields'] = fields
-        attrs['__slots__'] = fields.values()
         attrs['_instances'] = weakref.WeakValueDictionary()
+        attrs['__slots__'] = ['_hash'] + fields.values()
 
-        for base in bases:
-            if '__weakref__' in getattr(base, '__slots__', []):
+        anncestors = [b for base in bases for b in inspect.getmro(base)]
```
Comment (jodal) [inline mopidy/models.py]: s/anncestors/ancestors/

File: tests/models/test_models.py [hunk 2]
```diff
@@ -8,6 +8,17 @@
     TlTrack, Track, model_json_decoder)
 
 
+class InheritanecTest(unittest.TestCase):
```
Comment (jodal) [inline tests/models/test_models.py]: s/Inheritanec/Inheritance/

**Comment by jodal** [comment]
Looks good, except the typos.

**Comment by adamcik** [comment]
Fixed.

**Pull request merged**